dll/shellext/netshell/lang/zh-TW.rc

```diff
@@ -50,8 +51,8 @@ BEGIN
     LTEXT "位元組:", -1, 22, 115, 32, 8
     LTEXT "000.000.000", IDC_SEND, 60, 115, 44, 8
     RTEXT "000.000.000", IDC_RECEIVED, 146, 115, 44, 8
-    PUSHBUTTON "屬性(&P)", IDC_STATUS_PROPERTIES, 10, 150, 60, 14
-    PUSHBUTTON "禁用(&D)", IDC_ENDISABLE, 76, 150, 60, 14
+    PUSHBUTTON "內容(&P)", IDC_STATUS_PROPERTIES, 10, 150, 60, 14
```

Comment:
may I ask why translate “Properties” as 「内容」？

Reply:
(Sorry for the late reply because I received the comment in midnight)
This can be a tough questions to answer, because 'Properties' can be translated into either「屬性」 or 「內容」 in Traditional Chinese according to Mic**t terminology. (Please note that the uses of word 「內容」 is far more than 「屬性」in Traditional Chinese)
Therefore, I have compared the corresponding dialog with the Windows 2000 one to check whether the translation is fine.

This is the dialog in React
![dialog in React](https://user-images.githubusercontent.com/48596398/90468828-e7709a00-e149-11ea-9cd8-51967484c15b.png)

This is the equivalent dialog in Windows 2000 (I hope it is)
![dialog in Windows 2000](https://user-images.githubusercontent.com/48596398/90468976-3f0f0580-e14a-11ea-95a5-7592e72bba5b.png)

You can see that translation of two pushbuttons are different. To reduce confusion, I have made the translate “Properties” as 「内容」. Of course, it is fine for translate it as 「屬性」. I can revert the change if you want.

PS: The font in React is quite broken lol

Reply:
It makes sense🤣 Thank you for explain this. Possibly I should learn more about traditional Chinese.

> PS: The font in React is quite broken lol

yes 😅

> PPS: Is it possible for me to join lang-chinese translation team?

I think it's OK! But It's not in my power🤪. You can contact @ColinFinck  or some other admins for this